### DISCUSSION TITLE
Designer can't load on vs 2019

### DISCUSSION THREAD

**Discussion by XcantloadX** [AvaloniaUI/Avalonia#8772] [Q&A] [answered]
I'm able to debug the project, but when I open the designer I got this error
```
10:44:03.761 [Information]  Avalonia Package initialized
10:44:11.595 [Error]  No executable found
```
and the message "The designer is loading".
My project targets .net4.8.

**Comment by maxkatz6** [comment, marked as answer]
From what I remember only netcore executables were supported. @kekekeks correct me if I am wrong
